kubernetes-cronjob/massdriver.yaml

```diff
@@ -5,71 +5,191 @@ source_url: github.com/YOUR_ORG/<md .Name md>
 access: private
 type: application
 
+app:
+  envs:
+    # Use jq expressions to build environment variables from input params or connections
+    LOG_LEVEL: .params.configuration.log_level
+    # MONGO_DSN: .connections.mongo_authentication.data.authentication.username + ":" + .connections.mongo_authentication.data.authentication.password + "@" + .connections.mongo_authentication.data.authentication.hostname + ":" + (.connections.mongo_authentication.data.authentication.port|tostring)
+  policies: []
+    # Use jq expressions to select policies from artifact security blocks
+    # - .connections.sqs.data.security.iam.subscribe
+
 params:
+  required:
+    - namespace
+    - image
+    - resourceRequests
+    - job
   properties:
-    name:
-      type: string
     namespace:
-      type: string
+      title: Kubernetes Namespace
+      description: Application will be deployed into this namespace. If the namespace doesn't exist, it will be created.
+      $ref: https://raw.githubusercontent.com/massdriver-cloud/artifact-definitions/main/definitions/types/k8s-resource-name.json
       default: default
     image:
+      title: Container Image
       type: object
+      required:
+        - repository
+        - tag
       properties:
         repository:
-          type: string
-        sha:
+          $md.immutable: true
+          title: Repository
+          # Container repositories can be created or imported into massdriver at: https://app.massdriver.cloud/container-repositories
```

Comment:
This note is out of date. We don’t support them Any more in the UI. Docs have a CI/cd section 

https://docs.massdriver.cloud/applications/container-repositories/aws